services/management/dbaas/version_service_client.go

```diff
@@ -158,6 +166,38 @@ func (c *VersionServiceClient) Matrix(ctx context.Context, params componentsPara
 	return &vsResponse, nil
 }
 
+func (c *VersionServiceClient) RecommendedComponentVersion(ctx context.Context, product, component string) (string, *componentVersion, error) {
```

Comment:
let's add a comment

Reply:
I think shouldn't return latest version, only latest recommended version

Reply:
The version "number" is being returned because inside the `component` struct there is no field for it.
As part of the process of auto creating a `name` because it wasn't provided by the user when calling the API, I am appending the version number to the number like: `pxc-`+8.0.21+`cluster_name`